src/bda/plone/orders/mailnotify.py

```diff
@@ -453,7 +454,7 @@ def __call__(self, *args):
         for item_attrs in items:
             title = item_attrs['title']
             remaining_stock = item_attrs['remaining_stock_available']
-            item_out_of_stock_text = "%s (Remaining stock: %s)\n" %(title, remaining_stock)
+            item_out_of_stock_text += "%s (Remaining stock: %s)\n" %(title, remaining_stock)
```

Comment:
This needs fix. item_out_of_stock_text shoud be items_out_of_stock_text instead.
Will commit the fix in the next push.